x-pack/plugins/security_solution/public/app/deep_links/index.ts

```diff
@@ -409,6 +409,28 @@ export function updateGlobalNavigation({
             : AppNavLinkStatus.hidden,
           searchable: capabilities.siem.read_cases === true,
         };
+      case SecurityPageName.detections:
+        return {
+          ...link,
+          deepLinks:
+            link.deepLinks != null
+              ? [
+                  ...link.deepLinks.map((detLink) => {
+                    if (detLink.id !== SecurityPageName.alerts) {
```

Comment:
nit -> I will prefer to see `if (detLink.id === SecurityPageName.alerts)` and then have the logic inside

Reply:
👍🏽  updated a4e10f5